hw/ip/dma/data/dma.hjson

```diff
@@ -448,6 +462,10 @@
               name: "SHA512"
               desc: "Perform inline hashing using SHA512."
             }
+            { value: "4",
+              name: "MEMSET"
```

Comment:
I am repeating myself here, I'm sure, but I always lean towards having separate read/write enables to achieve 'read only', 'write only' and 'copy' functionality rather than programming a specific opcode into the hardware block, because this offers some additional flexibility.
For example, at the moment we have a bizarre situation where it is not possible to use the DMA controller to read a block of memory and compute SHA digests on the data without also writing the data out to some destination.